runtime/marshal.js

```diff
@@ -434,7 +438,7 @@ var caml_output_val = function (){
         if (v[0] == 255) {
           // Int64
           if (memo(v)) return;
-          writer.write (8, 0x12 /*cst.CODE_CUSTOM*/);
+          writer.write (8, 0x19 /*cst.CODE_CUSTOM_FIXED*/);
```

Comment:
We should still use CODE_CUSTOM when using older ocaml compilers. One can achieve that with an helper function: 
```javascript
//Provides: marshall_header_int64
//Version: < 4.08
function marshall_header_int64 (){ return 0x12 }
//Provides: marshall_header_int64
//Version: >= 4.08
function marshall_header_int64 (){ return 0x19 }
```